plugin/mru.vim

```diff
@@ -314,9 +322,14 @@ function! s:MRU_Open_File_In_Tab(fname, esc_fname)
 	    " Goto the tab containing the file
 	    exe 'tabnext ' . i
 	else
-	    " Open a new tab as the last tab page
-	    tablast
-	    exe 'tabnew ' . a:esc_fname
+		if (winnr("$") == 1) && (bufname("") == "") && !&l:modified && g:MRU_Open_File_Reuse_Current_Empty_Tab
```

Comment:
Can you remove the check for g:MRU_Open_File_Reuse_Current_Empty_Tab and make this the default behavior?

The l: prefix is not needed for modified. Can you remove the l: prefix?

Reply:
Are you suggesting that we can get rid of the `MRU_Open_File_Reuse_Current_Empty_Tab` option altogether and make it as default (as if MRU_Open_File_Reuse_Current_Empty_Tab was set to 1 in my current code)?
I would like that.
Let me know. I will update my code and the PR.

Reply:
Yes. We don't need the MRU_Open_File_Reuse_Current_Empty_Tab option. We can make this behavior the default behavior.